### PR TITLE
Implementing alerts and doing some refactoring.

### LINKED ISSUE #363: Implementation of Alerts
- Initial Research 

Example [link](https://docs.timescale.com/timescaledb/latest/tutorials/grafana/setup-alerts/#summary)

As an initial step we can try to send alter to Slack and Gmail for:
- participation rate under 100% for X period of time
- X account balance under Y number

Once we have the setup working, we will need to brainstorm and agreed on the proper alerts setup.

### PR DESCRIPTION
**Pull request by nnedkov-limechain**
**Detailed description**:
Moving everything related for monitoring to folder monitoring. And changing the participation rate to start at value 100.

**Which issue(s) this PR fixes**:
Fixes #363

**Special notes for your reviewer**:

**Checklist**
- [x] Documentation added
- [ ] Tests updated

### CODE REVIEW

File: app/process/handler/message/handler.go [hunk 1]
```diff
@@ -64,6 +64,8 @@ func NewHandler(
 			Name: constants.ValidatorsParticipationRateGaugeName,
 			Help: constants.ValidatorsParticipationRateGaugeHelp,
 		})
+		// Set to 100 in order not to fire alerts until a transaction is processed
+		participationRate.Set(100)
```
Comment (svetlio8) [inline app/process/handler/message/handler.go:68]: Maybe we can have the number as a constant?

Comment (nnedkov-limechain) [reply app/process/handler/message/handler.go:68]: yep good idea

**Review by svetlio8 — COMMENTED** [review]
Can we delete these files:
`examples/.DS_Store`
`examples/three-validators/.DS_Store`

**Comment by nnedkov-limechain** [comment]
> 

done :)

**Review by failfmi — APPROVED** [review]

**Review by svetlio8 — APPROVED** [review]

**Pull request merged**